### ISSUE TITLE
Python aid for Newbies

### ISSUE THREAD

**Issue by Satyam26-09** [Kushal997-das/Project-Guidance#228]
## 📚 Documentation

Being new to python language one can face difficulties with remote libraries. I would love to add installation command for few of them in beginning of: 

- Capture Image
- Generation-Sam-Music-Player
- Track Phone number location
                of **Project-Guidance\Desktop Application\Basic\Python**

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES

**Comment by Kushal997-das** [comment]
@Satyam26-09 Yes proceed but make sure there shouldn't be any duplicate project .
